test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -98,8 +103,16 @@ def parse_apstat(self, options='-rn'):
                 break
             index = 0
             obj = {}
-            for val in line.split():
-                obj[keys[index]] = val
+            line = line.split()
+            for i in range(0, len(line)):
+                obj[keys[index]] = line[i]
+                if keys[index] == 'State':
+                    obj[keys[index]] = line[i] + "  " + line[i + 1]
```

Comment:
On the last pass through this list, if keys[index] == 'Stale', you will raise a ValueError because you are doing i+1.

Reply:
I did not get a ValueError when I ran my unit test. There are still line[i] beyond keys[index] == 'State'.